android/quest/build.gradle

```diff
@@ -24,8 +24,8 @@ android {
         applicationId "org.smartregister.fhircore"
         minSdkVersion sdk_versions.min_sdk
         targetSdkVersion sdk_versions.target_sdk
-        versionCode 4
-        versionName "0.0.4"
+        versionCode 1
```

Comment:
can we remove this entirely? seems like no since it's referenced, but not sure how it's meaningful, is it supposed to match the the version name, but as an integer?